testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -379,6 +379,25 @@ def test_wildcard_terminal_selection(self, universe):
         ag_wild = universe.select_atoms('resname AS*')
         assert ag == ag_wild
 
+    def test_wildcard_single_selection(self, universe):
```

Comment:
Just a suggestion here, since these four tests have the same structure, you might want to squash them together using pytest.mark.parametrize.

Reply:
Sorry for the dealy in providing more details here. This is the sort of construct I would suggest going for:

```
@pytest.mark.parametrize('selstring, wildstring', [
    ('resname TYR THR', 'resname T?R'),
    ('resname ASN ASP HSD', 'resname *S?'),
    ('resname LEU LYS', 'resname L**'),
    ('resname MET', 'resname *M*')
])
def test_wildcard_selection(self, universe, selstring, wildstring):
    ag = universe.select_atoms(selstring)
    ag_wild = universe.select_atoms(wildstring)
    assert ag == ag_wild
```

Note: I've changed the `'resname T?R'` case to match https://github.com/MDAnalysis/mdanalysis/pull/2551#discussion_r384398177